db/seeds.rb

```diff
@@ -5,3 +5,54 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+require 'factory_bot_rails'
+require 'faker'
+
+# Clear existing data
+InvoiceItem.destroy_all
+Transaction.destroy_all
+Invoice.destroy_all
+Customer.destroy_all
+Item.destroy_all
+Merchant.destroy_all
```

Comment:
just wondering why you included these destroy_all? would we not want keep all of our csv imported data to view in the rails s development environment?